config/olm/openshift/0.2.0/manifests/dynatrace-routing_rbac.authorization.k8s.io_v1_rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  creationTimestamp: null
+  name: dynatrace-routing
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: dynatrace-routing
+subjects:
+- kind: ServiceAccount
+  name: dynatrace-routing
+  namespace: dynatrace
```

Comment:
> "dynatrace" namespace

I presume this doesn't work if the Operator gets installed on other namespaces. Actually, I think all of these roles/role bindings/service accounts shouldn't have their own files.

Reply:
Should we move them into the CSV instead?